rfc/seo.md

```diff
@@ -0,0 +1,72 @@
+- Landing pages might not be a good idea after all. Thin content used to be good SEO practice (have multiple pages for each thing), but this does not have lots of advantages to users so now Google penalizes this. As a rule of thumb, what's good for users will be good for rankings since Google uses machine learning on user behavior. So we should watch things like time on page, bounce rate etc. identify low performing pages vs high performing pages and perhaps remove or improve low performing pages.
+
+![https://moz.imgix.net/learn/guides/Moz-BGSEO-Ch4-01-outlines-01.svg?auto=format&ch=Width&fit=max&q=50&s=e236ac8928a1cd1940243f15ce7dbb6d](https://moz.imgix.net/learn/guides/Moz-BGSEO-Ch4-01-outlines-01.svg?auto=format&ch=Width&fit=max&q=50&s=e236ac8928a1cd1940243f15ce7dbb6d)
+
+- According to the poing above, user engagement becomes one of the most important things on a website. Which means adding the comment section is a good idea.
+
+- Keywords are important but sometimes the most authoritative keywords are not the best ones. High-volume keywords are also more competitive to rank higher (keyword difficulty). Carefully [exploring the long tail](https://moz.com/blog/long-tail-seo-target-low-volume-keywords-whiteboard-friday) of relevant keywords to us might offer us a better advantage in ranking and in financial conversions.
+
+  > For example, a person searching for "shoes" is probably just browsing. On the other hand, someone searching for "best price red womens size 7 running shoe" practically has their wallet out!
+
+- Perhaps we were not intentional enough concerning keywords in our current content. According to the point above, phrases formulated as questions are SEO gold. We could do a search on the most common questions on our space and make sure these questions are phrased in our content
+
+- Writing content is HARD so we need to make sure we are making the most for our buck. We need a strategy before deciding which content to write about. I've seen these two content strategies which seem very interesting
+
+  - 10x content (😂 geez I *hate* the name)
+    - Search the keyword(s) you want your page to rank for
+    - Identify which pages are ranking highly for those keywords
+    - Determine what qualities those pages possess
+    - Create content that’s better than that
+  - Top content refurbishment
+    - See which of your content performs better
+    - Take that content and [refurbish it for other platforms](https://moz.com/blog/refurbishing-top-content-whiteboard-friday) (Slideshare, YouTube, Instagram, Quora, etc.) to expand acquisition funnel beyond Google.
+
+- Page speed is a very important SEO metric so we need to be extremely careful when making the website dynamic. Perhaps it's best to never migrate the guides to be dynamic? But images are the usually the top 1 culprit in page speed. We should consider optmizing the images for mobile and compressing all the images we use.
```

Comment:
Good to track! Since deployment guide content never changes, we can either always serve it statically _or_ put a CDN in front of the website with a 24h cache, which is likely the fastest of all.

Reply:
CDN would be a great call, yes